pull_request_template.md

```diff
@@ -1,5 +1,30 @@
-Please ensure the following when opening a PR:
-- [ ] Added or updated tests
-- [ ] Updated documentation
-- [ ] Applied style guidelines
-- [ ] Considered generalizability beyond our own use case
+### Purpose
+_Please link to the issue being fixed by this PR:_
+
+- No issue is linked to this PR.
+---
+### Changes
+_Please list out what major changes were made in this PR to address the issue:_
+
+- No changes.
+---
+### Review Instructions
+_Please provide instructions about how should a reviewer test/verify the changes in this PR:_
+
+- No instructions.
+
+---
+### PR Checklist
+_Please ensure the following when opening a PR:_
+
+- [ ] This PR added or updated tests.
+- [ ] This PR updated docstrings or documentation.
+- [ ] Let Jodi (**jodihir**) know when there is user-facing documentation updates needed!
+- [ ] This PR applied (WDL/Python) style guidelines.
```

Comment:
I think this checklist should also include an item for "This PR has no WDL linting errors reported by Circle CI". Marcus added a WDL linting tool to skylab called `miniwdl` that will log any style errors in the circle CI tests. It won't make the test fail if there are errors, so I think we should mention it in this checklist.

Reply:
Also including a link our WDL style guidelines would be helpful!